consensus/misc/eip1559/eip1559.go

```diff
@@ -132,6 +132,53 @@ func ValidateHoloceneExtraData(extra []byte) error {
 	return ValidateHolocene1559Params(extra[1:])
 }
 
+// DecodeMinBaseFeeExtraData decodes the extraData parameters from the encoded form defined here:
+// https://github.com/ethereum-optimism/design-docs/blob/main/protocol/minimum-base-fee.md#minimum-base-fee-in-block-header
+//
+// Returns 0,0,0 if the format is invalid, though ValidateMinBaseFeeExtraData should be used instead of this function for
+// validity checking.
+func DecodeMinBaseFeeExtraData(extra []byte) (uint64, uint64, uint8) {
+	// Best effort to decode the extraData for every block in the chain's history,
+	// including blocks before the minimum base fee feature was enabled.
```

Comment:
Bit confused why we need to do this, as it seems you only ever call this for blocks which have the feature enabled? "Best effort" just seems a little wooly for consensus code so would love to tighten this up. 

The invalid case could return an error or even panic, if we haven't specified what to do in that case.

Reply:
I see, thanks for the context. I'll think on it some more.